crates/turbopack-cli/src/main.rs

```diff
@@ -1,28 +1,101 @@
 #![feature(future_join)]
 #![feature(min_specialization)]
 
-use anyhow::Result;
+use std::path::Path;
+
+use anyhow::{Context, Result};
 use clap::Parser;
+use once_cell::sync::Lazy;
 use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt, EnvFilter, Registry};
 use turbopack_cli::{arguments::Arguments, register};
+use turbopack_cli_utils::{exit::exit_guard, raw_trace::RawTraceLayer, trace_writer::TraceWriter};
 
 #[global_allocator]
 static ALLOC: turbo_tasks_malloc::TurboMalloc = turbo_tasks_malloc::TurboMalloc;
 
+static TRACING_OVERVIEW_TARGETS: Lazy<Vec<&str>> =
+    Lazy::new(|| vec!["turbo_tasks_fs=info", "turbopack_dev_server=info"]);
+static TRACING_TURBOPACK_TARGETS: Lazy<Vec<&str>> = Lazy::new(|| {
+    [
+        &TRACING_OVERVIEW_TARGETS[..],
+        &[
+            "turbopack=trace",
+            "turbopack_core=trace",
+            "turbopack_ecmascript=trace",
+            "turbopack_css=trace",
+            "turbopack_dev=trace",
+            "turbopack_image=trace",
+            "turbopack_dev_server=trace",
+            "turbopack_json=trace",
+            "turbopack_mdx=trace",
+            "turbopack_node=trace",
+            "turbopack_static=trace",
+            "turbopack_cli_utils=trace",
+            "turbopack_cli=trace",
+            "turbopack_ecmascript=trace",
```

Comment:
Seems like forgetting to add a new crate to this list is an easy oversight. There should either be a comment in `Cargo.toml` to remind people to do this, or we could read directly from `Cargo.toml` at compile-time. Or there might be some other way to do this automatically for all `turbopack*` crates?